Dockerfile

```diff
@@ -27,14 +28,19 @@ RUN cargo build -p sqld --release
 
 # runtime
 FROM debian:bullseye-slim
-COPY --from=builder /target/release/sqld /bin/sqld
+
+EXPOSE 5001 8080
+VOLUME [ "/var/lib/sqld" ]
+
 RUN groupadd --system --gid 666 sqld
 RUN adduser --system --home /var/lib/sqld --uid 666 --gid 666 sqld
-RUN apt-get update && apt-get install -y ca-certificates
-COPY docker-entrypoint.sh /usr/local/bin
-ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
-VOLUME [ "/var/lib/sqld" ]
 WORKDIR /var/lib/sqld
 USER sqld
-EXPOSE 5001 8080
+
+COPY docker-entrypoint.sh /usr/local/bin
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
It seems that reverse order would be better since docker-entrypoint.sh probably changes more often than ca-certificates.

Also don't you need to add `apt-get install -y ca-certificates` to builder stage?
I think it would be better to add `apt-get install -y ca-certificates` to chef stage and copy certificates from there so that changed dependencies wouldn't trigger reinstallation of certificates.

Reply:
> It seems that reverse order would be better since docker-entrypoint.sh probably changes more often than ca-certificates.

That is true, but the entrypoint changes pretty rarely, and when that happens we'll only save a layer that weights the size of `ca-certificates.crt`. I preferred to keep it tidy by grouping together `COPY`s from the same place. 

Let me know if you still want me to reorder.


>Also don't you need to add apt-get install -y ca-certificates to builder stage?
I think it would be better to add apt-get install -y ca-certificates to chef stage and copy certificates from there so that changed dependencies wouldn't trigger reinstallation of certificates.

We don't have to because our [Rust base image](https://github.com/rust-lang/docker-rust/blob/97a7686492c0ef95717f3f5e93ee24186a407329/1.73.0/buster/slim/Dockerfile) already does it.

Reply:
Thanks for the explanation about the rust base image containing ca-certificates.
Regarding the order. I'm fine with whatever order you pick. Just please name your intend correctly. If the commit was saying "reorder the layers to make them more readable at the expense of build speed" I wouldn't even add such comment. Saying that you're speeding something up when in fact you're doing the opposite is confusing.